crates/bevy_pbr/src/render/pbr_fragment.wgsl

```diff
@@ -107,9 +107,23 @@ fn pbr_input_from_standard_material(
     if ((pbr_bindings::material.flags & pbr_types::STANDARD_MATERIAL_FLAGS_BASE_COLOR_TEXTURE_BIT) != 0u) {
 #ifdef MESHLET_MESH_MATERIAL_PASS
         pbr_input.material.base_color *= textureSampleGrad(pbr_bindings::base_color_texture, pbr_bindings::base_color_sampler, uv, in.ddx_uv, in.ddy_uv);
-#else
+#else // MESHLET_MESH_MATERIAL_PASS
         pbr_input.material.base_color *= textureSampleBias(pbr_bindings::base_color_texture, pbr_bindings::base_color_sampler, uv, view.mip_bias);
-#endif
+
+#ifdef ALPHA_TO_COVERAGE
+    // Sharpen alpha edges.
+    //
+    // https://bgolus.medium.com/anti-aliased-alpha-test-the-esoteric-alpha-to-coverage-8b177335ae4f
+    let alpha_mode = pbr_bindings::material.flags &
+        pbr_types::STANDARD_MATERIAL_FLAGS_ALPHA_MODE_RESERVED_BITS;
+    if alpha_mode == pbr_types::STANDARD_MATERIAL_FLAGS_ALPHA_MODE_ALPHA_TO_COVERAGE {
+        pbr_input.material.base_color.a = (pbr_input.material.base_color.a -
+                pbr_bindings::material.alpha_cutoff) /
+                max(fwidth(pbr_input.material.base_color.a), 0.0001) + 0.5;
+    }
+#endif // ALPHA_TO_COVERAGE
+
+#endif // MESHLET_MESH_MATERIAL_PASS
```

Comment:
Is a2c not compatible with meshlets?

Reply:
I don't know enough about meshlets to know. I know that Nanite is incompatible with transparency. I'd rather not touch the meshlet code as it's being heavily worked on.

Reply:
I moved the code out of the meshlet block.